### PR TITLE
Checkout management

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
## Feat:
1. Add to cart 
2. add to wishlist 
3. implement price box

<a href="https://deploy-preview-5--shoe-zone-ecom.netlify.app/product">preview</a>

https://user-images.githubusercontent.com/60725346/161134346-ef8354c7-cd30-483c-a688-f71205fb2133.mov

### CODE REVIEW

File: src/pages/authPage/join.js [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+  return (
+    <div>
+      <Navbar />
+      <main>
+        <div className="form-container">
+          <div className="form-info">
+            <p className="heading-x-lg text-center">Join-us</p>
+            <div className="form-input">
+              <div className="input-box">
+                <label for="input" className="text-medium">
```
Comment (krgarima) [inline src/pages/authPage/join.js:14]: In react we use `htmlFor` instead of `for`

**Review by krgarima — CHANGES_REQUESTED** [review]
Suggestions:
1. When you add the same item in the cart, instead of item appearing as 2 different items, there should be only 1 item with quantity 2 and so on.
aq

2. Same item should not appear in wishlist more than once.
aw

3. Add a badge to wishlist and cart icon showing the number of items present in Cart and Wishlist.

**Comment by harshitpaliwal95** [comment]
> Suggestions:
> 
> 1. When you add the same item in the cart, instead of item appearing as 2 different items, there should be only 1 item with quantity 2 and so on.
>    aq
> 2. Same item should not appear in wishlist more than once.
>    aw
> 3. Add a badge to wishlist and cart icon showing the number of items present in Cart and Wishlist.

I will start working on it

**Review by Asuraking1n — APPROVED** [review]
Please add a button to apply dummy credentials, it make easy for us to check the project,
Codes looks good, some changes was already pointed out by others pls correct them.

**Comment by harshitpaliwal95** [comment]
> Suggestions:
> 
>     1. When you add the same item in the cart, instead of item appearing as 2 different items, there should be only 1 item with quantity 2 and so on.
>        aq
> 
>     2. Same item should not appear in wishlist more than once.
>        aw
> 
>     3. Add a badge to wishlist and cart icon showing the number of items present in Cart and Wishlist.

its resolve now

**Pull request merged**